content/en/data_jobs/databricks.md

```diff
@@ -137,6 +139,39 @@ Optionally, you can also set other init script parameters and Datadog environmen
 
 {{< /tabs >}}
 
+### Advanced settings (optional)
+
+1. set `DD_RUN_NAME` environment variable using `spark_env_vars` field for your one-time job run.
+
+   For Databricks Jobs submitted via the [one-time run API endpoint][8], set this variable to the same value as your request payload's `run_name`. This allows Data Jobs Monitoring to group your Jobs with the same `run_name` as one row in the Cluster table, showing aggregated infrastructure metrics across runs instead of one row per submitted run.
```

Comment:
```suggestion
   For Databricks Jobs submitted via the [one-time run API endpoint][8] (common when doing orchestration with tools outside of Databricks such as Airflow, Azure Data Factory), set this variable to the same value as your request payload's `run_name`. This allows Data Jobs Monitoring to aggregate your job runs and cluster utilization with the same `run_name` and show performance across time.
```

Reply:
>This allows Data Jobs Monitoring to aggregate **your job runs** and cluster utilization with the same `run_name` and show performance across time.

This is not correct – Job runs are aggregated based on the service / job_name which is from the `"run_name": "Example Job",` field. This will allow cluster metrics to be more correctly aggregated for these jobs, though.

So the primary realm of impact is how cluster metrics are rolled up.

Reply:
To clarify my comment a bit:

One time job runs are shown based on span.service which is also the span.job_name. The name comes from the [`run_name` field](https://docs.databricks.com/api/workspace/jobs/submit#run_name) of the POST body. 

DD_RUN_NAME will add a new tag, `run_name`, to cluster metrics. (Note this is different from the flaky `runname` which Databricks adds to cluster metrics). The web-ui changes are to group cluster metrics by the `run_name` tag when the tag is on the metrics.

Reply:
accept @warrierr 's suggested modification with minor tweak to incorporate the fact the runs are not aggregated by the `run_name`. Let me know if the current version make sense or not. thanks!